src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -126,7 +126,9 @@ export default class SourceFileLinter {
 				isUIComponent: this.isUi5ClassDeclaration(node, "sap/ui/core/UIComponent"),
 			});
 		} else if (
-			ts.isPropertyDeclaration(node) && node.name.getText() === "metadata" &&
+			ts.isPropertyDeclaration(node) &&
```

Comment:
isn't the check for "static" missing here?

Reply:
Yep, thanks. Done.